packages/dd-trace/src/opentracing/tracer.js

```diff
@@ -8,6 +8,8 @@ const SpanContext = require('./span_context')
 const Writer = require('../writer')
 const Recorder = require('../recorder')
 const Sampler = require('../sampler')
+const AgentExporter = require('../agent/exporter')
```

Comment:
I would prefer to keep exporters together, for example `exporters/agent.js` and `exporters/log.js`.